app/views/map/_indoor.html.erb

```diff
@@ -3,13 +3,33 @@
         "fillOpacity": 0.0,
         "color": "#444"
     };
+    let roomLayer;
+    let roomTooltip;
     <% @buildings.each do |building|%>
         <% building.rooms.each do |room|%>
+
+            roomLayer = L.geoJSON(<%= raw(JSON.generate(room.to_geojson)) %>, 
+            {style: indoorStyle});
+            roomTooltip = L.tooltip({
+                permanent: true, 
+                interactive: true, 
+                className: 'marker_label', 
+                offset: L.point(0,0), 
+                direction: 'center',
+            });
+            roomTooltip.setContent("<%= room.full_name %>");
```

Comment:
If I am zoomed out, I see:
![image](https://user-images.githubusercontent.com/62988232/148639025-c3678789-a17a-42a2-aafa-6f6562d63675.png)
but if I'm zoomed in I see:
![image](https://user-images.githubusercontent.com/62988232/148639033-1d546891-a0aa-4099-a5b8-a88b8f875652.png)

It has to be the opposite.

Reply:
we did not deal with the zoom in this issue as https://github.com/hpi-swt2/compass-portal-red/issues/137 will deal with the display of the indoor data, including the room names.

Reply:
update: having thought about it: at the moment, you can either see all names (including buildings and rooms) or none. if we switched this at this high level, you wouldn't be able to see the names of the buildings when zoomed out. is that really expected?
and splitting the display of room and building names up is actually a bit of work, that's why my call would be dealing with it in #137 and leaving it in as it is right now.

Reply:
Alright